docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+# SPDX-FileCopyrightText: 2006-2023 Open Knowledge Foundation and contributors
+# SPDX-FileContributor: PNED G.I.E.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+version: "3"
+
+volumes:
+  pg_data:
+
+services:
+  postgres:
+    build:
+      context: postgresql/
+    environment:
+      - POSTGRES_USER
+      - POSTGRES_PASSWORD
+      - POSTGRES_DB
+      - KC_DB_USERNAME
+      - KC_DB_PASSWORD
+      - KC_DB_NAME
+      - REMS_DB_USER
+      - REMS_DB_PASSWORD
+      - REMS_DB
+    volumes:
+      - pg_data:/var/lib/postgresql/data
+    restart: unless-stopped
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "${POSTGRES_USER}", "-d", "${POSTGRES_DB}"]
+    ports:
+      - "5432:5432"
+
+  keycloak:
```

Comment:
Is curious if it is the best way to have separate docker compose for each of the service. With this we will end up with multiple instances of keycloak no?

There will be one in ckan as well.

Isn't there a way where we could just have 1 docker-compose for all readymade services like (postgres, keycloak, rems, ckan) and then spawn our inhouse services like this one separately?

I'm thinking on the lines how hard it is right now to run the whole thing locally end to end

Reply:
Indeed we have a single repo for the end2end deployment. https://github.com/GenomicDataInfrastructure/gdi-userportal-deployment

I added here just because there is no keycloak + rems deployment for manual end2end testing.